utils/cams_wind_pipeline.py

```
@@ -1,167 +1,114 @@
 import pandas as pd
+import xarray as xr
 from sqlalchemy import create_engine, MetaData, Table, Column, Float, DateTime, Index
 from sqlalchemy.exc import SQLAlchemyError
 from dotenv import load_dotenv
-from pathlib import Path
-import cftime
 import logging
-import tempfile
-import shutil
+import glob
+from typing import Optional
```

Ruff (0.11.9)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)